content/en/tracing/troubleshooting/_index.md

```diff
@@ -164,12 +164,14 @@ var tracer = Tracer.Create(isDebugEnabled: true);
 Tracer.Instance = tracer;
 ```
 
-Location of the profiler log:
+The environment variable `DD_TRACE_DEBUG` can also be set to `true`.
+
+Logs (Profiler and Tracer) are saved in the following directories:
 
 | Platform | Path                                                          |
 |----------|---------------------------------------------------------------|
-| Linux    | `/var/log/datadog/dotnet-profiler.log`                        |
-| Windows  | `C:\ProgramData\Datadog .NET Tracer\logs\dotnet-profiler.log` |
+| Linux    | `/var/log/datadog/`                        |
+| Windows  | `C:\ProgramData\Datadog .NET Tracer\logs\` |
```

Comment:
We should use the `%ProgramData%` variable here. In theory it could be something different than `C:\ProgramData`.

```suggestion
| Windows  | `%ProgramData%\Datadog .NET Tracer\logs\` |
```